reference/edit-window/metadata/extent-panels/features-import.md

```diff
@@ -14,6 +14,13 @@ __Drag & Drop__  The map supports dragging a file from your file explorer and dr
 
 ---
 
+{% hint style='info' %}
+**Please Note:**  Imported data needs to utilize the widely used World Geodetic System 1984 (WGS 84) [WGS84] coordinate reference system.  This has longitude and latitude set in units of decimal degrees. For more details on this, please visit <a href="https://epsg.io/4326" target="_blank">the espg.io coordinate support site</a>.  Here you can find details for the defined unit measurements as well as popular file export options.
```

Comment:
```suggestion
**Please Note:**  Imported data needs to utilize the widely used World Geodetic System 1984 (WGS 84) [WGS84] coordinate reference system.  This has longitude and latitude set in units of decimal degrees. For more details on this, please visit <a href="https://epsg.io/4326" target="_blank">the espg.io database website</a>.
```

Reply:
See #24

Reply:
@IronRico I'll let you handle these inline suggestions - don't want to commit and mess up your workflow.

Reply:
Didn't test to see if that `<sup>` will render correctly in the markdown. If not just remove it. We should add some css for `<a class="external">` to visually indicate an external link opening a new tab. Example:

![image](https://user-images.githubusercontent.com/5833603/86163469-bfa37500-babc-11ea-8be3-779933d94167.png)